app/src/main/java/com/quebec/VideoPreview.java

```diff
@@ -0,0 +1,89 @@
+package com.quebec;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.provider.MediaStore;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.VideoView;
+
+public class VideoPreview extends AppCompatActivity {
```

Comment:
Refactor the name of the class to be VideoPreviewActivity so that it is consistent with other activities.